### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by paweltalar**

### CODE REVIEW

File: documentation/API_documentation/DeviceLocation_API.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+<img src="./resources/Location_overview.png" alt="LOC_VER" title="Location verification">
+
+## 2\. Quick Start
+Location API v0.1.0 exposes only one endpoint, which can be used to determine if user equipment is within the given radial distance from
+given location. To do so we need to pass following parameters in request body: 
+1. **ueId** - an object with four fields, each of them make possible to pass ueId in different format: externalId [1], msisdn, ipv4Addr and ipv6Addr 
+2. **latitude** - a floating point number representing latitude of location, value from -90 to 90
+3. **longitude** - a floating point number representing longitude of location, value from -180 to 180
+4. **accuracy** - an integer representing accuracy (radius) expected for location verification in km
+
```
Comment (ukkaps) [inline documentation/API_documentation/DeviceLocation_API.md:16]: Have introduced another parameter to the list, pls update the documentation accordingly.

https://github.com/camaraproject/DeviceLocation/pull/8

**Review by ukkaps — COMMENTED** [review]
Pls. incorporate.

File: documentation/API_documentation/DeviceLocation_API.md (hunk 1 above)
Comment (jlurien) [reply documentation/API_documentation/DeviceLocation_API.md:16]: First, we have to approve the PR

**Comment by jlurien** [comment]
I'm closing this one as documentation is to be updated along with any new functionality

**Pull request closed without merging**